src/components/item/form-variety.svelte

```diff
@@ -1,44 +1,61 @@
 <script>
+  import sizes from '@/constants/item/sizes';
   import Button from 'ui/button.svelte';
   import TextField from 'ui/text-field.svelte';
   import Dropdown from 'ui/dropdown.svelte';
-  import {item, changeVarietyQuantity, changeVarietySize, removeVariety} from '@/store/item';
+  import RadioGroup from 'ui/radio-group.svelte';
+  import DragDropzone from '@/components/item/drag-dropzone.svelte';
+  import {item, changeVarietySize, removeVariety, changeVarietyField, colors, customColors, addNewColor} from '@/store/item';
 
   export let index;
   export let removable;
-  export const colors = [];
 
-  const sizes = ['XS', 'S', 'M', 'L' , 'XL', 'XXL'];
-  const variety = $item.varieties[index];
+  let colorPicker = null;
+
+  $: variety = $item.varieties[index];
+  const chooseColor = e => {
+    changeVarietyField(index, 'color', e.detail.color);
+  };
+
+  const openColorPicker = () => colorPicker.click();
```

Comment:
```suggestion
  const openColorPicker = colorPicker.click;
```
(which doesn't need an extra function, imo)

Reply:
You have just wrote the assignment of `null` variable `colorPicker`'s undefined property `click`. Constant function is needed for imitating the click on the input to open the color picker.

Reply:
Ah, I didn't notice this. Sorry